internal/newusers/cache/db.go

```diff
@@ -221,34 +200,31 @@ func (c *Cache) Close() error {
 	return c.db.Close()
 }
 
+// Clear removes the database file.
+func Clear(cacheDir string) error {
```

Comment:
It’s not really clearing here, we are actually removing the database. So I would name this one `Remove` (as your comment intented)